src/window/file_dialog.c

```diff
@@ -78,25 +81,64 @@ static input_box file_name_input = {144, 80, 20, 2, FONT_NORMAL_WHITE, 0, data.t
 static file_type_data saved_game_data = {"sav"};
 static file_type_data scenario_data = {"map"};
 
-static int double_click = 0;
+static int find_first_file_with_prefix(const char *prefix)
+{
+    int len = strlen(prefix);
```

Comment:
```suggestion
    size_t len = strlen(prefix);
```

Please change otherwise MSVC throws a warning.